app/views/transactions/_filter.html.slim

```diff
@@ -13,5 +18,6 @@ h4 Filters:
     .col-sm-6
       = f.input :date_to, input_html: { class: 'datepicker' },
         placeholder: 'dd/mm/yyyy', label: 'To:'
-  .pull-right
+  .pull-right.btn-group
+    = f.button :submit, 'Reset', type: 'reset', class: 'btn-default'
```

Comment:
@Gataniel I think it will be better to use
`f.button :button, 'Reset', :type => 'reset', :class => 'btn btn-default'`